packages/radix-vue/src/Accordion/story/AccordionSingle.story.vue

```diff
@@ -104,11 +104,11 @@ const selectedValue = ref("item-2");
       </AccordionRoot>
     </Variant>
 
-    <Variant title="Collapsible">
+    <Variant title="Not Collapsible">
       <AccordionRoot
         class="w-[300px] rounded-md bg-[--line-color] shadow-lg"
         type="single"
-        :collapsible="true"
+        :collapsible="false"
```

Comment:
I think let's revert this back to `Collapsible`. As the default value for `collapsible` is `false`. this Variant here allow us to test `collapsible: true`

Reply:
True, my bad